src/plugins/intel_gpu/src/graph/impls/ocl/fully_connected.cpp

```diff
@@ -105,6 +135,17 @@ struct fully_connected_impl : typed_primitive_impl_ocl<fully_connected> {
         auto optional_params = get_default_weights_bias_optional_params<kernel_selector::fully_connected_optional_params>(progam);
         optional_params.allowInputReordering = true;
 
+        bool commpressed = !primitive->compression_scale.empty();
+        bool with_zp = !primitive->compression_zero_point.empty();
+        if (commpressed) {
+            params.compressed = true;
+            params.decompression_scale = convert_data_tensor(input_layouts[2]);
```

Comment:
it was `compression_scale` in another place. What about using same name, either `compression` or `decompression`?

Reply:
changed to decompression_scale/zero_point